### DISCUSSION TITLE
Proposal for GPU support for image processing algorithms

### DISCUSSION THREAD

**Discussion by codejaeger** [JuliaImages/Images.jl#945] [JSoC]
I am Debabrata Mandal a CS undergrad from India. I find the project "Adding GPU support for image processing algorithms" quite interesting to work on. Still quite new to Julia, I will be exploring both JuliaGPU and JuliaImages before I come up with a substantial plan for how to proceed. I just had a few questions though:

1. Will the benchmarking project (also proposed for GSoC) have any direct dependence on this project?
2. The projects involving creating new packages seem very interesting but given the changed GSoC plan are they put on hold or we can still make a proposal for them?

Thanks.

**Comment by timholy** [comment]
Welcome! To address your questions:
1. There isn't any obligate dependence on the benchmarking project; in an ideal world we'd already have benchmarks in place so it would be easy for you to measure the improvements in performance.
2. I don't think there would be any new packages needed to do the GPU project, but we'd have to see. Or are you referring to some of the other project ideas? You'd of course just pick one, you do not have to do them all.

**Comment by codejaeger** [comment]
Yes, I was referring to the other 2 projects below:
1. https://julialang.org/jsoc/gsoc/images/#contributions_to_a_stereo_matching_package_medium
2. https://julialang.org/jsoc/gsoc/images/#contributions_to_a_calibration_target_package_medium

Thanks @timholy, you have answered my question anyways.
